lib/gvl-tracing.rb

```diff
@@ -35,6 +35,7 @@ class << self
     private :_stop
 
     def start(file)
+      _init_local_storage(Thread.list)
       _start(file)
```

Comment:
Thanks for applying my previous suggestion!

I've been thinking -- should we reorder these two lines? E.g. should we start listening to events, and then initialize the threads? This way if any threads get created concurrently, we'll go ahead and create contexts for them, and for existing threads we do that anyway.

The only downside I see is that we may miss a few events from the existing threads right at the beginning, but arguably that seems OK since they were anyway concurrent with `_start`.

Reply:
I swapped those, not sure of the consequences but it seems to work fine for the examples where the threads are either created before or after tracing starts, but not concurrently.